### DISCUSSION TITLE
OAuth2: revoking access tokens

### DISCUSSION THREAD

**Discussion by Nielsvanpach** [saloonphp/saloon#375] [Ideas]
Hi Sam,

First of all, thank you for this package! Working with Saloon makes it way easier to develop integrations.

Unless I missed it, I couldn't find any information regarding the revocation of access tokens. 
Is there anything in place yet for this?

Thank you for your time and attention!
Niels

Example: _https://developers.linear.app/docs/oauth/authentication#id-6.-revoke-an-access-token_

**Comment by Sammyjo20** [comment]
Hey @Nielsvanpach! 

You're very welcome - thank you for the kind words. I noticed that you work for Spatie? May I ask if you are using it internally or is it just for personal projects? (It'll be so cool if you guys are using it!)

Unfortunately no, revocation of access tokens is not yet part of the Saloon OAuth integration - however, I would be happy to note it down as a future feature, or happy to accept a PR to add the functionality.

Best regards
Sam
